src/components/Footer.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState, useContext } from 'react';
+import { Status } from '../services/Status';
+import { TodosContext } from './TodosContext';
+
+export const Footer: React.FC = () => {
+  const {
+    todos,
+    clearForm,
+    setFilter,
+    filter,
+  } = useContext(TodosContext);
+  const [count, setCount] = useState(0);
+
+  // Функція для оновлення кількості невиконаних завдань
```

Comment:
The comments in the code are not in English. It's a good practice to write comments in English so that it can be understood by all developers.